### PR TITLE
@alloy => Initial import of danger

### PR DESCRIPTION
**Pull request by orta**

### CODE REVIEW

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1 @@
+puts "Hello"
```
Comment (alloy) [inline Dangerfile]: Does this actually do something yet, despite this stub file?

Comment (orta) [inline Dangerfile]: yeah, it _should_ be ran need to verify the environment variables

Comment (alloy) [inline Dangerfile]: Ah ok, I see, it’s still a WIP :+1:

File: Gemfile [hunk 2]
```diff
@@ -16,6 +16,7 @@ group :test do
   gem 'fui'
   gem 'xcpretty'
   gem 'second_curtain'
+  gem 'danger', :git => "https://github.com/KrauseFx/danger.git"
```
Comment (KrauseFx) [inline Gemfile]: :rocket:

**Comment by ArtsyMobile** [comment]
:white_check_mark: | No errors found
  ------------- | ------------ 



  :white_check_mark: | No warnings found
  ------------- | ------------ 




<p align="right">
  Generated by 
    <a href="https://github.com/KrauseFx/danger">danger</a>
  on 
  <i>2016-01-07</i>
</p>

**Comment by orta** [comment]
OK, assuming this greens. I'm good for this.

**Comment by orta** [comment]
I'm happy for the Dangerfile to grow and evolve naturally, rather than try to anticipate all our plans.

**Comment by orta** [comment]
Fixes: https://github.com/artsy/mobile/issues/31

**Comment by ArtsyMobile** [comment]
&nbsp; | 2 Errors
  ------------- | ------------ 
    :no_entry_sign: | No CHANGELOG changes made
    :no_entry_sign: | Needs testing on a Phone if change is non-trivial



  :white_check_mark: | No warnings found
  ------------- | ------------ 




<p align="right">
  Generated by 
    <a href="https://github.com/KrauseFx/danger">danger</a>
  on 
  <i>2016-01-07</i>
</p>

**Comment by orta** [comment]
Oh yeah, needs a changelog entry, thanks Danger!

**Comment by ArtsyMobile** [comment]
:white_check_mark: | No errors found
  ------------- | ------------ 



  &nbsp; | 1 Warning
  ------------- | ------------ 
    :warning: | Needs testing on a Phone if change is non-trivial




<p align="right">
  Generated by 
    <a href="https://github.com/KrauseFx/danger">danger</a>
  on 
  <i>2016-01-07</i>
</p>

**Comment by orta** [comment]
:shipit:

**Comment by orta** [comment]
Great choice of GIFs.

**Comment by KrauseFx** [comment]
So beautiful :heart:

**Comment by alloy** [comment]
Is quoting Austin Powers ever _not_ a good idea?

Groovy, baby!

**Comment by ashfurrow** [comment]
I like this so much

**Pull request merged**